### PR TITLE
:fire: [New Term]: C++ Math-Functions: `.tanh()` (#981)

### LINKED ISSUE #981: [Term Entry] C++ Math Functions: `tanh()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `tanh()` function.

- In the terms/ folder, make a new folder called `tanh`
- Inside the `tanh` folder, create a new markdown file called `tanh.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by tanishq-singh-2301**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #981 in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#include <iostream>
+#include <cmath>
+
+int main()
+{
+  double value = 1.0;
+  double result = std::tanh(value);
+
+  std::cout << "The value of tanh(" << value << ") is: " << result << "\n";
+}
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:82]:
```suggestion
int main() {
  double value = 1.0;
  double result = std::tanh(value);

  std::cout << "The value of tanh(" << value << ") is: " << result << "\n";
}
```

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Example
+
+The following example demonstrates the application of `.tanh()` function:
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main()
+{
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:39]:
```suggestion
int main() {
```

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`Math.tanh()`** function returns the hyperbolic tangent of an `double`-type arguement given in radians.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:14]:
```suggestion
The **`Math.tanh()`** function returns the hyperbolic tangent of an `double`-type argument given in radians.
```

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+```
+
+The `angle` is measured in radians and the return type is a `double`, `float`, or `long double`.
+
+The mathematical formula used in `tanh()` looks like this:
+
+```tex
+tanh(x) = (e^x - e^-x) / (e^x + e^-x)
+```
+
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:29]:
```suggestion

tanh(x) = (e<sup>x</sup> - e<sup>-x</sup>) / (e<sup>x</sup> + e<sup>-x</sup>)

```

While the \```tex blocks [are technically supported](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#latex) in our content, they haven't been rendering as expected in our Docs entries (e.g., [here](https://www.codecademy.com/resources/docs/cpp/math-functions/cosh)).

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@tanishq-singh-2301 This entry is already off to a great start! I just had a few comments/suggestions from my review. 😄

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`Math.tanh()`** function returns the hyperbolic tangent of an `double`-type argument given in radians.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:14]:
```suggestion
The **`Math.tanh()`** function returns the hyperbolic tangent of an angle given in radians.
```

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
+---
+Title: 'tanh()'
+Description: 'Returns the hyperbolic tangent of a double-type value.'
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:3]:
```suggestion
Description: 'Returns the hyperbolic tangent of an angle given in radians.'
```

File: content/cpp/concepts/math-functions/terms/tanh/tanh.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+std::tanh(angle)
+```
+
+The `angle` is measured in radians and the return type is a `double`, `float`, or `long double`.
+
+The mathematical formula used in `tanh()` looks like this:
+
+```tex
+tanh(x) = (e<sup>x</sup> - e<sup>-x</sup>) / (e<sup>x</sup> + e<sup>-x</sup>)
+```
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/tanh/tanh.md:28]: @tanishq-singh-2301 Unfortunately, we aren't able to combine \```tex code with GH-flavored HTML (`<sup></sup>`). We have two options:

- Remove the \```tex backticks and keep `<sup></sup>`. (recommended)
- Remove `<sup></sup>` and keep \``` tex code block.
  - Again, this is not likely to render correctly at this time.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@tanishq-singh-2301 Just a few more comments, and then I'll approve 😄

**Review by Dusch4593 — APPROVED** [review]
@tanishq-singh-2301 LGTM for the next review! 👍🏻

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by tanishq-singh-2301** [comment]
any update? @Dusch4593 @HishamT

**Comment by Dusch4593** [comment]
Congrats and thank you for another solid contribution to Codecademy Docs, @tanishq-singh-2301! 🎊 🎉 🥳

The entry should now be live: https://www.codecademy.com/resources/docs/cpp/math-functions/tanh

Screen Shot 2022-11-21 at 9 46 24 AM

**Pull request merged**